packages/react-native-codegen/src/parsers/flow/modules/index.js

```diff
@@ -228,6 +228,27 @@ function translateTypeAnnotation(
       }
     }
     case 'ObjectTypeAnnotation': {
+      // if there is any indexer, then it is a dictionary
+      if (typeAnnotation.indexers) {
+        const indexers = typeAnnotation.indexers.filter(
+          member => member.type === 'ObjectTypeIndexer',
+        );
+        if (indexers.length > 0) {
+          // check the property type to prevent developers from using unsupported types
+          const propertyType = indexers[0].value;
+          translateTypeAnnotation(
```

Comment:
We are not using the returned value here, is it correct?

Reply:
Same